financials/Funding request template.txt

```diff
@@ -0,0 +1,24 @@
+## Template for funding request

```

Comment:
Maybe this could be made into a unique template title? Like so:
`Funding Request: [insert your name], [insert funding period]`